test/lit/passes/intrinsic-lowering.wast

```diff
@@ -0,0 +1,42 @@
+;; NOTE: Assertions have been generated by update_lit_checks.py and should not be edited.
+;; RUN: wasm-opt %s --intrinsic-lowering -all -S -o - | filecheck %s
+
+(module
+  ;; call.if.used with no params.
+  ;; CHECK:      (import "binaryen-intrinsics" "call.if.used" (func $ciu-v (param funcref) (result i32)))
+  (import "binaryen-intrinsics" "call.if.used" (func $ciu-v (param funcref) (result i32)))
+
+  ;; call.if.used with some params.
+  ;; CHECK:      (import "binaryen-intrinsics" "call.if.used" (func $ciu-dif (param f64 i32 funcref) (result f32)))
+  (import "binaryen-intrinsics" "call.if.used" (func $ciu-dif (param f64) (param i32) (param funcref) (result f32)))
```

Comment:
Are we able to import the same module/base combination multiple times with different signatures..? Didn't know that...

Reply:
Yes, it's a little odd, but it's useful in rare cases like when importing something from JS that we can pick different types for.